registry/quilt_server/prod_config.py

```diff
@@ -8,31 +8,21 @@
 SQLALCHEMY_DATABASE_URI = os.environ['SQLALCHEMY_DATABASE_URI']
 
 REGISTRY_HOST = os.environ['REGISTRY_HOST']
-OAUTH_HOST = os.environ['OAUTH_HOST']
-OAUTH_API_HOST = os.environ['OAUTH_API_HOST']
+REGISTRY_URL = 'https://{REGISTRY_HOST}'.format(REGISTRY_HOST=REGISTRY_HOST)
```

Comment:
This will make it hard for anyone who wants to run a private registry without https. That's arguably a bad idea, but I don't think we need to enforce https here.

Reply:
What would you suggest doing? Make a separate environment variable REGISTRY_URL? Or what?